Interface Improvements

BurgerLUA opened the pull request:
# Overview
Improves the general interface to look less windows 2000 and adds some additional settings, while removing useless ones.

![Wew](https://i.imgur.com/0Ao7UPm.png)

BurgerLUA commented:
!review

HeyBanditoz commented on interface/skin.dmf:
```diff
@@ -1423,168 +1011,69 @@ window "Telecomms IDE"
 		type = MAIN
 		pos = 281,0
-		size = 640x440
+		size = 640x480
```
?????????¿¿¿¿¡

BurgerLUA replied:
what the fuck does this even mean?

HeyBanditoz replied:
You tell me.

skull132 commented:
Neither the PR nor the changelog provide any insight into _what_ was actually changed. Please actually describe your modifications.

BurgerLUA commented:
Changes:
- Removed Text Mode, which just made the screen black.
- Made the default map zoom 64x64.
- Added sprite scaling settings for the menus.
- Removed the borders for most of the elements.
- Made the horizontal and vertical slider togglable.
- Recolored the chat bar and buttons.
- Modified the font of the chat bar to be bigger.

BurgerLUA commented:
The chat isn't bigger. That's just my personal setting after tinkering with the vertical scroll bar. When implemented, the split will be it's original 50/50.

Lohikar commented:
Oh, I misread. Font size of the chat _bar_. Thought you were increasing the font size of chat in general.
